shopfloor/tests/test_cluster_picking_scan_line.py

```diff
@@ -288,7 +340,9 @@ def test_scan_line_location_error_several_products(self):
         # Although, if one line is already processed,
         # we can work automatically on the next one
         line.qty_done = line.product_uom_qty
-        self._scan_line_ok(new_move.move_line_ids[0], line.location_id.barcode)
+        line._action_done()
+        # FIXME the previous action_done does not work !
```

Comment:
FIXME

Reply:
Same remark, we should not call `_action_done` on a line, this one is called by the `stock.move._action_done()`, itself called by `stock.picking._action_done()` (which is the only method we should deal with when it comes to validate a move).